services/network/network.go

```diff
@@ -961,24 +961,20 @@ type IPTag struct {
 	Tag *string `json:"tag,omitempty"`
 }
 
+const DefaultIdleTimeoutInMinutes uint32 = 4
+
 // PublicIPAddressProperties public IP address properties.
 type PublicIPAddressPropertiesFormat struct {
+	// IPAddress - The IP address associated with the public IP address resource.
+	IPAddress *string `json:"ipAddress,omitempty"`
 	// PublicIPAllocationMethod - The public IP address allocation method. Possible values include: 'Static', 'Dynamic'
 	PublicIPAllocationMethod IPAllocationMethod `json:"publicIPAllocationMethod,omitempty"`
 	// PublicIPAddressVersion - The public IP address version. Possible values include: 'IPv4', 'IPv6'
 	PublicIPAddressVersion IPVersion `json:"publicIPAddressVersion,omitempty"`
 	// IPConfiguration - READ-ONLY; The IP configuration associated with the public IP address.
 	IPConfiguration *IPConfiguration `json:"ipConfiguration,omitempty"`
-	// DNSSettings - The FQDN of the DNS record associated with the public IP address.
-	DNSSettings *PublicIPAddressDNSSettings `json:"dnsSettings,omitempty"`
-	// IPTags - The list of tags associated with the public IP address.
-	IPTags *[]IPTag `json:"ipTags,omitempty"`
-	// IPAddress - The IP address associated with the public IP address resource.
-	IPAddress *string `json:"ipAddress,omitempty"`
-	// PublicIPPrefix - The Public IP Prefix this Public IP Address should be allocated from.
-	PublicIPPrefix *SubResource `json:"publicIPPrefix,omitempty"`
 	// IdleTimeoutInMinutes - The idle timeout of the public IP address.
-	IdleTimeoutInMinutes *int32 `json:"idleTimeoutInMinutes,omitempty"`
+	IdleTimeoutInMinutes *uint32 `json:"idleTimeoutInMinutes,omitempty"`
```

Comment:
>uint32 [](http://example.com/codeflow?start=23&length=6)

this too, can you please make sure we are aligned with azure?

Reply:
int32 to unit 32. There is no way for IdleTimeoutInMinutes can be negative.